docs/source/contents/4_draft_mapping.rst

```diff
@@ -668,6 +670,195 @@ All values within <tableOfContents> will be mapped to RDF in the same way. Below
+
+    <name authority="naf" valueURI="http://id.loc.gov/authorities/names/no2002022963">
+        <namePart>Swan, W. H. (William H.)</namePart>
+        <role>
+            <roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/cmp">
+                Composer
+            </roleTerm>
+        </role>
+    </name>
+
+Leverage the valueURI and make it the object of the triple:
+
+.. code-block:: turtle
+    :caption: Resulting RDF `from harp:1 <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_
+    :name: Resulting RDF `from harp:1 MODS <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cmp <http://id.loc.gov/authorities/names/no2002022963> .
+
+When there is no :code:`mods:name/@valueURI`, use the string literal from :code:`mods:name/mods:namePart`:
+
+.. code-block:: xml
+    :caption: XML with Name missing a valueURI
+    :name: XML with Name missing a valueURI
+
+    <name type="personal">
+        <namePart>Daniel, Charles R. (Charlie), Jr., 1930-</namePart>
+        <role>
+            <roleTerm type="text" authority="marcrelator" valueURI=" http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+.. code-block:: turtle
+    :caption: Resulting turtle for Name missing a valueURI
+    :name: Resulting turtle for Name missing a valueURI
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cre "Daniel, Charles R. (Charlie), Jr., 1930-" .
+
+If there is a :code:`mods:name/valueURI` but it's empty, use the string literal instead:
+
+.. code-block:: xml
+    :caption: Example XML from `volvoices:2495 MODS <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+    :name: Example XML from `volvoices:2495 MODS <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+
+    <name authority="naf" type="corporate" valueURI="">
+        <namePart>Bemis Bro. Bag Company</namePart>
+        <role>
+            <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/asn">Associated name</roleTerm>
+        </role>
+    </name>
+
+.. code-block:: turtle
+    :caption: Resulting turtle from `volvoices:2495 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+    :name: Resulting turtle from `volvoices:2495 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:asn "Bemis Bro. Bag Company" .
+
+Keep all Roles
```

Comment:
Can we rename this use case to describe the situation better? Maybe just "Names with multiple roleTerms".

Reply:
:+1: